src/v/pandaproxy/rest/configuration.cc

```diff
@@ -54,6 +54,13 @@ configuration::configuration()
       "consumer_instance_timeout_ms",
       "How long to wait for an idle consumer before removing it",
       {},
-      std::chrono::minutes{5}) {}
+      std::chrono::minutes{5})
+  , rest_require_auth(
+      *this,
+      "rest_require_auth",
```

Comment:
I think I'd prefer if this was spelled out `authentication`, to prevent confusion with `authorization`